src/components/influencer-details/InfluencerDetails.js

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
 import _ from "lodash";
 
-import {userAvatar} from "../../utils/avatar";
+import { convertNumbers } from "../../utils/convertNumbers";
```

Comment:
could you pls ensure that we have the same thing on top mentioned trolls section as well